test/e2e/run_env_test.go

```diff
@@ -90,7 +90,15 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run --env-host environment test", func() {
-		SkipIfRemote("FIXME, We should check that --env-host reports correct error on podman-remote")
+		if IsRemote() {
+			env := append(os.Environ(), "FOO=BAR")
+			session := podmanTest.PodmanAsUser([]string{"run", "--rm", "--env-host", ALPINE, "/bin/printenv", "FOO"}, 0, 0, "", env)
```

Comment:
Similar to above, can we eliminate some duplication?

Reply:
Fixed